### DISCUSSION TITLE
Allowing to generate an address {domain}-{random string} in browser addon

### DISCUSSION THREAD

**Discussion by theo-jolivel** [simple-login/app#457] [Feature Requests]
I love to have an address for each website based on the domain, like github@domain.tld.
But I think that it is too predictable and you could be spammed quite easily using the most visited websites.

I know that I can do it manually but I would love to have the option in the browser addon to generate an alias based on the domain plus a random short string, for instance github-fef43ns@domain.tld to make it unpredictable.

Thanks for this great work !

**Comment by nguyenkims** [comment]
There's this option "Random Prefix Generation" on the custom domain page that will automatically add the random suffix as you described :). This option can be set on the web version at https://app.simplelogin.io/ .

![image](https://user-images.githubusercontent.com/3167780/116914702-598efd80-ac4b-11eb-8fa7-9f1a7eef4948.png)

**Reply by theo-jolivel** [reply]
Oh ok I didn't see that, thanks !
